example/rnWebRTC/android/build.gradle

```diff
@@ -0,0 +1,34 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        buildToolsVersion = "28.0.3"
+        minSdkVersion = 16
```

Comment:
really ?

Reply:
I think the following versions of Android 24 are not supported or I couldn't run

Reply:
It must be something else. We use 19 with no problems. 16 is indeed too old, Google no longer supports it.

Reply:
16 is the minimum version needed by ReactNative itself, newer versions would be needed by specific native modules if they use more recent APIs, but they would need to specify it in their docs.

Reply:
I get this error when I make minSDK 23 in the project.
![Uploading Ekran Resmi 2020-03-04 11.36.39.png…]()

Reply:
Try adding this to your gradle.properties file: android.enableDexingArtifactTransform.desugaring=false

Reply:
I am trying...

Reply:
YES it is worked! Thanks for support. Can you write about the "code" ?

Reply:
What "code" ?

Reply:
android.enableDexingArtifactTransform.desugaring=false

Reply:
What is the question? This disables some ccode transformations that seem to break WebRTC. It started to appear with the gradle plugin 3.5.0.

Reply:
Thanks